cassandra-analytics-core/src/test/java/org/apache/cassandra/spark/utils/CqlUtilsTest.java

```diff
@@ -259,10 +267,14 @@ public void testExtractTableSchemaCase2(CassandraBridge bridge)
                                   + "\"C2\" blob, "
                                   + "\"C4\" blob, "
                                   + "value counter, "
-                                  + "PRIMARY KEY (key, column1) ) WITH "
-                                  + "cdc = true AND max_index_interval = 2048 AND min_index_interval = 128;";
+                                  + "PRIMARY KEY (key, column1) ) WITH"
+                                  + " bloom_filter_fp_chance = 0.1"
+                                  + " AND cdc = true"
+                                  + " AND compression = {'chunk_length_in_kb': '64', 'class': 'org.apache.cassandra.io.compress.DeflateCompressor'}"
+                                  + " AND default_time_to_live = 100 AND max_index_interval = 2048 "
+                                  + "AND min_index_interval = 128;";
         String actualCreateStmt = CqlUtils.extractTableSchema(schemaStr, "keyspace", "table");
-        assert expectedCreateStmt.equals(actualCreateStmt);
```

Comment:
what! 👍

Reply:
This comment in isolation got me totally confused - thought I literally just removed an assert, and you gave it a 👍 😃